src/modules/add.js

```diff
@@ -62,5 +58,47 @@ textareaSelector.forEach((textarea) => {
     if (e.target.value) {
       updateText(e.target.previousElementSibling.id, e.target.value);
     }
+    if (e.target.previousElementSibling.checked) {
+      e.target.style.textDecoration = 'line-through';
+    }
+  });
+});
+
+const checkbox = document.querySelectorAll('input[type=checkbox]');
+
+for (let i = 0; i < tasks.length; i += 1) {
+  checkbox[i].addEventListener('change', (e) => {
+    tasks.forEach((task) => {
+      if (task.index === +e.target.id) {
+        if (e.target.checked) {
+          task.completed = true;
+          e.target.nextElementSibling.style.textDecoration = 'line-through';
+        } else {
+          task.completed = false;
+          e.target.nextElementSibling.style.textDecoration = 'none';
+        }
+      }
+      localStorage.setItem('savedData', JSON.stringify(tasks));
+    });
   });
+}
+
+document.addEventListener('DOMContentLoaded', () => {
+  for (let i = 0; i < tasks.length; i += 1) {
+    tasks.forEach((task) => {
+      if (+checkbox[i].id === task.index) {
+        if (task.completed === true) {
+          checkbox[i].checked = true;
+          // textareaSelector.forEach((textarea) => {
+          //   textarea.style.textDecoration = 'line-through';
+          // });
```

Comment:
### 🛑 Required Change
- [ ] Kindly delete commented out code. This helps in maintaining easy to read code